diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -155,26 +153,43 @@ class BlockStructureBraces : Rule("block-structure") {
             CLASS, OBJECT_DECLARATION -> node.findChildByType(CLASS_BODY)!!.findChildAfter(LBRACE, WHITE_SPACE)
             else -> node.findChildByType(BLOCK)?.findChildAfter(LBRACE, WHITE_SPACE)
         }
-        if (nodeText != null && !nodeText.text.contains("\n".toRegex())) {
-            BRACES_BLOCK_STRUCTURE_ERROR.warnAndFix(configRules, emitWarn, isFixMode, "incorrect line after opening brace",
-                    nodeText.startOffset) {
+        if (newNode != null && !newNode.text.contains("\n")) {
+            BRACES_BLOCK_STRUCTURE_ERROR.warnAndFix(configRules, emitWarn, isFixMode, "incorrect same line after opening brace",
+                    newNode.startOffset) {
+            }
+        }
+    }
+
+    private fun checkMidBrace(node: ASTNode){
+        val allMiddleSpace = when(node.elementType){
+            TRY -> {
+                node.findAllNodesWithSpecificType(FINALLY).map { it.prevSibling { type -> type.elementType == WHITE_SPACE } } +
+                node.findAllNodesWithSpecificType(CATCH).map { it.prevSibling { type -> type.elementType == WHITE_SPACE } }
+            }
+            else -> node.findAllNodesWithSpecificType(THEN).map { it.nextSibling { type -> type.elementType == WHITE_SPACE } }
```

Comment:
I suggest to specify condition explicitly: now your check works only for `TRY` and `IF`, if I understood correctly, and `else` branch should throw error. Then when this code will be modified in future it will be more clear, how to add handling of new type of nodes.